RFS/src/main/java/org/opensearch/migrations/bulkload/common/SnapshotCreator.java

```diff
@@ -23,22 +23,21 @@ public abstract class SnapshotCreator {
     private final IRfsContexts.ICreateSnapshotContext context;
     @Getter
     private final String snapshotName;
+    @Getter
+    private final String snapshotRepoName;
     private final List<String> indexAllowlist;
 
-    protected SnapshotCreator(String snapshotName, List<String> indexAllowlist, OpenSearchClient client,
-            IRfsContexts.ICreateSnapshotContext context) {
+    protected SnapshotCreator(String snapshotName, String snapshotRepoName, List<String> indexAllowlist,
+                              OpenSearchClient client, IRfsContexts.ICreateSnapshotContext context) {
         this.snapshotName = snapshotName;
+        this.snapshotRepoName = snapshotRepoName;
```

Comment:
Nit: if this was called repo name it would reduce the number of changes in this file.